cloud/edgecontroller/conf/controller.yaml

```diff
@@ -16,4 +16,11 @@ cloudhub:
   keepalive-interval: 30
   write-timeout: 30
   node-limit: 10
-
+devicecontroller:
```

Comment:
the edge controller also has a "kube" config, also device controller requires this config . could we have a separate config like  a global kube config which could be reused?

Reply:
+1 to the idea.

Reply:
Can there be a scenario where device-management is done on 1 k8s master setup and deployment management in another ? If no we can make entire kube configurations global.